src/main/kotlin/com/safehill/kclient/network/remote/RemoteServer.kt

```diff
@@ -547,7 +604,7 @@ class RemoteServer(
             .header(mapOf("Authorization" to "Bearer $bearerToken"))
             .body(
                 Gson().toJson(
```

Comment:
can we abandon the gson usage?

Reply:
I think so but why?

Reply:
Gson uses reflection for serialization. Once we enable R8, the variable names would get obfuscated and we would have to remember to keep the model classes using 'proguard keep rules' which is much more tedious than simply using kotlin serialization.